jobs/haproxy/templates/keepalived.config.erb

```diff
@@ -0,0 +1,35 @@
+global_defs {
+	# Keepalived process identifier
+	lvs_id <%= p('keepalived.lvs_id') %>
+}
+# Script used to check if HAProxy is running
+vrrp_script <%= p('keepalived.healthcheck_name') %> {
+	script "<%= p('keepalived.healthcheck_command') %>"
+	interval 2
+	weight 2
+}
+# Virtual interface
+# The priority specifies the order in which the assigned interface to take over in a failover
+vrrp_instance <%= p('keepalived.vrrp_instance_name') %> {
+	<% if p('keepalived.masterip').include? spec.networks.marshal_dump.first[1].ip %>
```

Comment:
Alternative 1 is not a proper solution because by default the release spawns 2 VMs in 2 different groups. According do BOSH documentation, spec.index works this way : 

> <% spec.bootstrap %>: Allow to evaluate if this instance is a the first instance of its group.
> <%= spec.index %>: Inserts instance index. Use spec.bootstrap instead of checking for index being 0. Additionally there is no guarantee that all instances won’t have gaps in their indices.

We could use : 
> <%= spec.name %>: Inserts instance name.

combined with
> <% spec.bootstrap %>: Allow to evaluate if this instance is a the first instance of its group.

To be sure that we won't have strange behaviour, we could ask BOSH operator to insert in it's deployment manifest an instance name that match for example a pattern like haproxy_master.

It will give us something like : 
```ruby
<% if p('spec.name').include? 'haproxy_master' && p('spec.bootstrap')? %>
#MASTER
<% else %>
#SLAVE
<% end %>
```
(Ruby code above not tested :p)